packages/react-avatar/package.json

```diff
@@ -42,6 +42,7 @@
   },
   "dependencies": {
     "@fluentui/react-badge": "^9.0.0-alpha.46",
+    "@fluentui/react-icons": "^1.1.130",
```

Comment:
Should we discuss how best to introduce this dependency in our weekly sync ? If each component has a separate dependency on this icon package, it might be difficult for our consumers to use different packages and figure out what icons are being used in their app ?

There could potentially be a case to use it as peer dependencies too perhaps

Reply:
> case to use it as peer dependencies too perhaps

peerDependencies sounds good to me! just keeps me wondering what approach would users need to take if they would like to use different set of icons? Provide them one by one per every component slots?

Reply:
> Should we discuss how best to introduce this dependency in our weekly sync ?

that sounds like a very good idea -> output from that conversation should be written down as an RFC probably

Reply:
@ling1726 @Hotell is it ok to check in this PR as-is, and when the dependencies issue is resolved, that can be handled in a future PR?

Reply:
sounds good @behowell